### PR TITLE
[TASK] Add test for automatically adding a missing DOCTYPE

### PR DESCRIPTION
**Pull request by oliverklee**
This bug fix slipped in with one of the latest refactorings. Now we
also have a unit test for it.

### CODE REVIEW

**Comment by oliverklee** [comment]
I've had this branch on my computer for some time now and would like to have it out of the way.

**Comment by oliverklee** [comment]
It looks like something still is missing. I'll have a look.

File: tests/Unit/EmogrifierTest.php [hunk 1]
```diff
@@ -267,6 +267,18 @@ public function emogrifyBodyContentKeepsSpecialCharacters($codeNotToBeChanged)
...
     }
 
+    /**
+     * @test
+     */
+    public function addsMissingHtml5DocumentType()
+    {
+        $this->subject->setHtml('<html></html>');
+
+        $result = $this->subject->emogrifyBodyContent();
```
Comment (zoliszabo) [inline tests/Unit/EmogrifierTest.php]: As far as I can see, the only issue (the cause of failed test) is that this line should read `$result = $this->subject->emogrify()`.

Comment (oliverklee) [reply tests/Unit/EmogrifierTest.php]: Oh, of course. D'oh!

**Comment by oliverklee** [comment]
Fixed and repushed.

**Comment by zoliszabo** [comment]
👍

**Pull request merged**